src/components/ManaCalculator/components/ManaCalculatorInput.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { JSX, useEffect, useState } from 'react';
+
+interface IManaCalculatorCustomInput {
+  value: string | number;
+  onChange: (value: string) => void;
+  className?: string;
+  min?: number;
+  max?: number;
+}
+
+export function ManaCalculatorInput({
```

Comment:
This component can be much more generic, its not really related to the mana calculator, is an input that also has an error message based on validators, it can be renamed and moved to a more generic place